TestTools/StreamChatTestMockServer/MockServer/MessageResponses.swift

```diff
@@ -534,9 +537,61 @@ public extension StreamMockServer {
         return .ok(.json(json))
     }
     
-    private func mockMessageReplies(_ messageId: String) -> HttpResponse {
+    private func limitReplies(_ request: HttpRequest) throws -> HttpResponse {
```

Comment:
I still think it makes sense to name this mock replies, instead of limitReplies, the limit pagination is an implementation detail in this case 
```suggestion
    private func mockMessageReplies(_ request: HttpRequest) throws -> HttpResponse {
```

Reply:
Done 👍